docs/Customs Declare Exports AutoComplete.js

```diff
@@ -374,38 +374,18 @@ function additionalActors(){
 }
 
 function holderOfAuthorisation(){
-    if (currentPageIs("/customs-declare-exports/declaration/holder-of-authorisation")) {
+    if (currentPageIs("/customs-declare-exports/declaration/add-authorisation-required")) {
```

Comment:
Please bump the version of QS.

I always forgot about it, don't know how I can remember it during review :D

Reply:
Done